.ci/basic_python_requirements

```diff
@@ -1,3 +1,3 @@
 sqlalchemy==1.0.9
 alembic==0.8.2
-thrift==0.9.1
+thrift==0.9.3
```

Comment:
0.9.1 can not be installed with python 3, but in the meantime I realized that python3 generators for thrift will only come for 1.0, so I will just revert this for now.